projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java

```diff
@@ -575,6 +575,35 @@ public String getObject(String containerName, String testrigName, String objectN
     }
   }
 
+  @Nullable
+  public JSONObject getQuestionTemplates() {
+    try {
+      WebTarget webTarget = getTarget(CoordConsts.SVC_RSC_GET_QUESTION_TEMPLATES);
+
+      MultiPart multiPart = new MultiPart();
+      multiPart.setMediaType(MediaType.MULTIPART_FORM_DATA_TYPE);
+
+      addTextMultiPart(multiPart, CoordConsts.SVC_KEY_API_KEY, _settings.getApiKey());
+
+      JSONObject jObj = postData(webTarget, multiPart);
+      if (jObj == null) {
+        return null;
+      }
+
+      if (!jObj.has(CoordConsts.SVC_KEY_QUESTION_LIST)) {
+        _logger.errorf("question list key not found in: %s\n", jObj.toString());
```

Comment:
drop `.toString()` -- `%s` should already handle that conversion.

(And better yet, also handles `null` values; tho in this case that's not an issue)